src/main/java/com/twilio/http/ValidationClient.java

```diff
@@ -84,7 +84,8 @@ public Response makeRequest(Request request) {
             HttpResponse response = client.execute(builder.build());
             return new Response(
                 response.getEntity() == null ? null : response.getEntity().getContent(),
-                response.getStatusLine().getStatusCode()
+                response.getStatusLine().getStatusCode(),
+                response.getAllHeaders()
```

Comment:
Where's the definition for `getAllHeaders`? I only see `getHeaders`

Reply:
https://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/org/apache/http/HttpMessage.html#getAllHeaders()

Reply:
Then what's the benefit of the `getHeaders` function?

Reply:
`getHeaders` requires a string argument for getting specific headers.

Reply:
[I don't see it taking in any arguments](https://github.com/twilio/twilio-java/pull/553/files/b43da0e4f047cf549bab5941d2387248012be505#diff-feb98a20878237b5c61d91e7f832f4f5R117)

Reply:
`HttpResponse` != `Response`